### PR TITLE
Improve container debugging experience

### PR DESCRIPTION
**Pull request by zhiltsov-max**
### Motivation and context

- Fixed `django.db.utils.operationalerror: server closed the connection unexpectedly` errors during debugging rq workers
- Added an option to disable request timeouts during debugging

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ services:
         CLAM_AV:
         CVAT_DEBUG_ENABLED:
     environment:
+      # Use this with CVAT_DEBUG_ENABLED to avoid server response timeouts
+      # Remove the same setting in the docker-compose.yml for this override to work
+      # DJANGO_MODWSGI_EXTRA_ARGS: '--request-timeout 0 --queue-timeout 0 --socket-timeout 1000000'
```
Comment (azhavoro) [inline docker-compose.dev.yml:25]: After upgrading to uvicorn these options are no longer available, I think that similar behavior can be achieved by changing `*timeout` options in [nginx configuration](https://github.com/opencv/cvat/blob/develop/cvat/nginx.conf), but need to check.

https://nginx.org/en/docs/http/ngx_http_proxy_module.html

Comment (zhiltsov-max) [reply docker-compose.dev.yml:25]: Ok, ASGI turned out to be working normally in debug without these settings. However there is no debug wrapper in the `asgi.py`, so I added one. Do we still need `cvat/wsgi.py`?

Comment (zhiltsov-max) [reply docker-compose.dev.yml:25]: Ok, ASGI turned out to be working normally in debug without these settings. However there is no debug wrapper in the `asgi.py`, so I added one. Do we still need `cvat/wsgi.py`?

Comment (azhavoro) [reply docker-compose.dev.yml]: We no longer need `cvat/wsgi.py`, it seems I forgot to delete the file in my PR

Comment (zhiltsov-max) [reply docker-compose.dev.yml]: Done

**Review by azhavoro — APPROVED** [review]

**Pull request merged**